MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceModel.cpp

```diff
@@ -560,6 +561,19 @@ void TomographyIfaceModel::refreshLocalJobsInfo() {
   }
 }
 
+void TomographyIfaceModel::updateProcessInJobList(const qint64 pid,
+                                                  const int exitCode) {
+  // cast to string from qint64 so we can compare
+  const std::string processPID = std::to_string(static_cast<int>(pid));
```

Comment:
I've removed the static_cast<int>, `to_string` seems to work with a qint64, because that is just a [typedef of an int](http://doc.qt.io/qt-4.8/qtglobal.html#qint64-typedef).

Just for fun:
- Max PID on Linux is 4194303, by default it's 32768 ([source](http://stackoverflow.com/questions/6294133/maximum-pid-in-linux))
- Max PID on Windows is DWORD_MAX, but it must be aligned to 4, so the max PID is 0xFFFFFFFC or 4294967292 ([source](http://stackoverflow.com/questions/17868218/what-is-the-maximum-process-id-on-windows))

So this was a potential very bad and uncaught `static_cast` failure on PID's > 2147483647.